blues/airnote.live
# bugs: fixes for a few css mobile issues

# Problem Context

While testing, I noticed the mobile menu didn't close on page change and the toast notifications were too big for the screen.

## Changes

* Add some CSS to make the toasts appropriately sized for screens under 600px wide
* Add a function to close the mobile menu if it is open after a page navigation in the app.

## Screenshot (if applicable)

https://github.com/blues/airnote.live/assets/20400845/5e80f941-f5ff-4acb-94fb-bbc08648fa00

## Testing

Unit / integration / e2e tests?
Steps to test manually?

1. Size down the viewport to under 600px
2. Go to the Airnote dashboard
3. see the mobile menu close after navigating to the dashboard page
4. Click the copy link button and see the success toast that is sized correctly for the screen

Any other sorts of testing notes to include?

## Any other related PRs

n/a

## Ticket(s)

n/a
